### PR TITLE
Wizard form

### PR DESCRIPTION
**Pull request by rawagner**
depends on https://github.com/openshift-assisted/assisted-ui-lib/pull/631

### CODE REVIEW

File: src/components/InfraEnv/InfraEnvForm.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+          <StackItem>
+            <Grid hasGutter span={8}>
+              <GridItem>
+                <Title headingLevel="h1" size={TitleSizes.xl}>
+                  Configure environment
+                </Title>
+              </GridItem>
+              <GridItem>
+                Infrastructure Environments are used by Clusters. Create an Infrastructure
+                Environment in order to add resources to your cluster
```
Comment (jtomasek) [inline src/components/InfraEnv/InfraEnvForm.tsx:125]:
```suggestion
                Environment in order to add resources to your cluster.
```

File: src/components/InfraEnv/InfraEnvForm.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                  <UploadSSH />
+                  <ProxyFields />
+                </Form>
+              </GridItem>
+            </Grid>
+          </StackItem>
+          {error && (
+            <StackItem>
+              <Alert
+                className="kv-create-vm__error"
```
Comment (jtomasek) [inline src/components/InfraEnv/InfraEnvForm.tsx:143]: leftover className?

**Review by jtomasek — COMMENTED** [review]
2 nits inline

File: src/components/InfraEnv/InfraEnvForm.tsx (hunk 2 above)
Comment (rawagner) [reply src/components/InfraEnv/InfraEnvForm.tsx]: good catch, removed

File: src/components/InfraEnv/InfraEnvForm.tsx (hunk 1 above)
Comment (rawagner) [reply src/components/InfraEnv/InfraEnvForm.tsx]: added

**Review by jtomasek — DISMISSED** [review]

**Review by jtomasek — APPROVED** [review]

**Pull request merged**